packages/esm-appointments-app/src/form/appointments-form.component.tsx

```diff
@@ -178,6 +178,19 @@ const AppointmentsForm: React.FC<AppointmentsFormProps> = ({
         path: ['appointmentDateTime.recurringPatternEndDate'],
         message: 'A recurring appointment should have an end date',
       },
+    )
+    .refine(
+      (formValues) => {
+        const appointmentDate = formValues.appointmentDateTime?.startDate;
+        const dateAppointmentScheduled = formValues.dateAppointmentScheduled;
+
+        if (!appointmentDate || !dateAppointmentScheduled) return true;
+        return dateAppointmentScheduled < appointmentDate;
+      },
+      {
+        path: ['dateAppointmentScheduled'],
+        message: 'Date appointment issued cannot be after the appointment date',
```

Comment:
Can we make this localizable using t()? Thanks!

Reply:
Done.

Thanks @mogoodrich . Thanks @hadijahkyampeire